examples/haitham_work_area/genz_110_palindrome_structure.py

```diff
@@ -0,0 +1,6 @@
+n = int(input('Enter the size of the palindrome structure: '))
+
+for i in range(1, n + 1):
+    print(' '*(n-i), end='')
+    print(''.join(str(j) for j in range(1, i)), end='')
```

Comment:
Please add single line comments to explain what each loop is doing because a dev would have to dry run to understand